contextual_lenses/pfam_family_classification/analyze_pfam_sweeps.py

```diff
@@ -0,0 +1,41 @@
+"""Analyze and plot pfam train samples sweeps."""
+
+
+import pandas as pd
+
+import matplotlib.pyplot as plt
+
+
+df = pd.read_csv('samples_sweep.csv')
```

Comment:
Put this and the calls to sweep_plot below inside a main function, and do

```
if __name__ == "__main__":
  ...
```

Good practice so that the code isn't executed if you, for example, import the file into ipython notebook then call the function (in which case you don't want this default case to run)

https://stackoverflow.com/questions/419163/what-does-if-name-main-do